sale_coupon_multi_currency/views/sale_coupon_views.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="sale_coupon_view_form" model="ir.ui.view">
+        <field name="name">sale.coupon.form.currency</field>
+        <field name="model">sale.coupon</field>
+        <field name="inherit_id" ref="sale_coupon.sale_coupon_view_form" />
+        <field name="arch" type="xml">
+            <field name="code" position="before">
+                <field name="currency_program_id" invisible="1" />
```

Comment:
because its only useful if you are using it for something. I added it for reusability. Whats the point to show it there, if its not gonna be related with any amount fields?